src/main/java/fruitshop/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,27 @@
+package fruitshop.service.impl;
+
+import fruitshop.model.FruitTransaction;
+import fruitshop.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    public static final int TYPE = 0;
+    public static final int FRUIT = 1;
+    public static final int QUANTITY = 2;
+    public static final String COMA = ",";
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputReport) {
+        List<FruitTransaction> result = new ArrayList<>();
+        for (String fruitReport : inputReport) {
```

Comment:
The method `convertToTransaction` assumes that every line of `inputReport` is correctly formatted. It's a good practice to handle potential formatting errors gracefully, by either skipping the malformed lines with a warning or throwing an informative exception.